tests/IntegrationTests/IntegrationTestService/IntegrationTestService.csproj

```diff
@@ -1,12 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1; net5.0</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>false</IsPackable>
```

Comment:
nit: can it still run on net 5? i don't remember.

Reply:
nevermind, i removed it, i just don't remember why right now. 

---
In reply to: [498561988](https://github.com/AzureAD/microsoft-identity-web/pull/636#discussion_r498561988) [](ancestors = 498561988)

Reply:
:)